src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -229,6 +230,11 @@ def app_teardown(
 @app.command("deploy", requires_connection=True)
 @with_project_definition("native_app")
 def app_deploy(
+    files: Optional[List[Path]] = typer.Argument(
+        default=None,
+        show_default=False,
+        help=f"""Paths of the files, relative to the deploy root, to be uploaded to a stage. If the path exists remotely but not locally, it will be deleted from the stage [default: sync all local changes to the stage]""",
```

Comment:
To be clear, are we specifying files relative to the deploy root or relative or CWD? Because deploy root != CWD. My vote is for the CWD since a user is less likely to remember where the file will be on the stage, but have more familiarity with their actual repo structure.

Reply:
I am rewriting this to be relative to project root (which is currently CWD)